plugins/webp-uploads/hooks.php

```diff
@@ -780,3 +780,31 @@ function webp_uploads_init(): void {
 	}
 }
 add_action( 'init', 'webp_uploads_init' );
+
+/**
+ * Automatically opt into extra image sizes when generating fallback images.
+ *
+ * @since n.e.x.t
```

Comment:
```suggestion
 * @since n.e.x.t
 *
 * @global array $_wp_additional_image_sizes Associative array of additional image sizes.
```

Add missing global doc.